### PR TITLE
Slime Mob Changes

### PR DESCRIPTION
**Pull request by Alekshhh**
## About the PR
Changes the slime mob damage and damage modifiers. Also changed Smile to be immune to rads and poison.

## Why / Balance
I wanted slimes to be more dangerous when encountered unprepared, and less of a threat when ready. Also Smile deserves to be rad immune.

## Technical details
Slimes now do more damage, and their damage type is now unique based on color, along with their existing blunt damage - green slimes primarily cause poison, yellow slimes cause genetic damage, and blue slimes cause caustic damage.
In turn, slimes take a lot more damage from water vapors, fire extinguishers in particular. It should take around three good uses to kill them
Green slimes are a bit stronger than the rest, due to poison damage being simpler to treat. They will take down someone in one hit less than the rest.
They also had their poison and cellular damage modifiers lowered, just felt right
Smile is immune to poison and radiation now, but they take more damage from cold damage, this also felt right.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:Alekshhh
- tweak: Changed slimes to hit harder, but crumple under fire extinguishers. Their damage type is more unique based on color now. Smile is now immune to radiation and poison as well.

### CODE REVIEW

File: Resources/Prototypes/Damage/modifier_sets.yml [hunk 1]
```diff
@@ -106,8 +106,8 @@
     Slash: 1.2
     Piercing: 1.2
     Cold: 1.5
-    Poison: 0.8
-    Cellular: 0.2
+    Poison: 0.4
+    Cellular: 0.0
```
Comment (mirrorcult) [inline Resources/Prototypes/Damage/modifier_sets.yml:110]: this applies to slimepeople too right? does it need to be changed

Comment (Alekshhh) [reply Resources/Prototypes/Damage/modifier_sets.yml:110]: don't tell me that's the case

Comment (Alekshhh) [reply Resources/Prototypes/Damage/modifier_sets.yml:110]: fucking hell it is

**Comment by Alekshhh** [comment]
god damn it hold on don't review it yet

**Comment by Alekshhh** [comment]
should be fine now I hope

**Comment by metalgearsloth** [comment]
Didn't we just nerf a bunch of simple mobs for wiping out stations?

**Comment by AJCM-git** [comment]
There really hasnt been a discussion on this and the changes are pretty arbitrary, probably better to make an issue with the problem this tries to solve and agree to a solution there

**Pull request closed without merging**